metrics/metrics.go

```diff
@@ -49,6 +49,12 @@ type PrivacyLabels struct {
 	LMTEnforced    bool
 }
 
+type ModuleLabels struct {
+	Module string
+	Stage  string
+	PubID  string // account ID
```

Comment:
The account ID will be captured during the future stages. The entrypoint stage does not have access to account information.
Regarding the labels, `ModuleLabels` is just a higher level DTO passed to different metric engines. We are not creating account ID labels for prometheus right because of the cardinality if you are concerned about that.
But the account ID will be used to store flat metrics for go-metric engine. This logic is already used in PBS-Go by account adapter metrics.

Yes, the PRD states that and PBS-Java indeed uses some kind of verbosity scale (none, basic, detailed) for all account metrics (not only module/hook related). But I did not find anything similar in PBS-Go and it seems that it records all account metrics without any verbosity checks, that's why I decided to follow this convention for account module metrics.
If you think that for account module metrics we should use the verbosity level, then we should be able to retrieve it from the account config.